lib/documents.js

```diff
@@ -140,10 +146,20 @@ export async function register({
     // 3. Attempt to refresh the expiration for the document if it is not
     //    known to be a new registration.
     if(!newRegistration &&
-      await _refresh({externalIdHash, documentHash, ttl, creatorHash})) {
+      await _refresh({
+        externalIdHash, documentHash, ttl, creatorHash,
+        minAssuranceForResolution
+      })) {
+      const {registration} =
+        await _getRegistrationRecord({externalIdHash, documentHash});
+      if(updateTokenBatch) {
+        await updateBatch({
```

Comment:
There might be more than one batch, right? Shouldn't this be pluralized and be changed to handle updating all open batches?

Reply:
Updated.